lib/commands.js

```diff
@@ -547,10 +548,13 @@ exports.bindCommands = function bindCommands(forum) {
                 if (!checkAvailable(forbiddenCmds, cmd, 'error', forbiddenMsg)) {
                     return reject(new Error('E_FORBIDDEN_COMMAND'));
                 }
+                if (context === undefined) {
+                    context = new Object();
```

Comment:
yep. from the checkin comment (and thus the changelog): 

Any command that relies on the previously undocumented value of `this` within a command handler will be broken by this update.